scapy_ssl_tls/ssl_tls.py

```diff
@@ -594,18 +594,46 @@ class TLSCertificateList(PacketNoPayload):
                    PacketListField("certificates", None, TLSCertificate, length_from=lambda x:x.length),
                   ]
 
-class TLSDigitallySigned(PacketNoPayload):
-    name = "TLS Digitally Signed"
-    fields_desc = [
-                   PacketField("algorithm", None, TLSSignatureHashAlgorithm),
-                   StrField("signature", "", fmt="H"),  # ASN.1 signature element
-                  ] 
 
 class TLSCertificateVerify(PacketNoPayload):
     name = "TLS Certificate Verify"
-    fields_desc = [
-                   PacketField("digitally-signed", None, TLSDigitallySigned),
-                  ]  
+    fields_desc = [PacketField("alg", None, TLSSignatureHashAlgorithm),
+                   XFieldLenField("sig_length", None, length_of="sig", fmt="H"),  # ASN.1 signature element
+                   StrLenField("sig", "", length_from=lambda x:x.sig_length)]
+
+
+TLS_CERTIFICATE_TYPE = {0x1: "rsa_sign",
```

Comment:
There is an enum with the shiny autogenerated name `TLS_CLIENTCERTIFICATETYPE_IDENTIFIERS_REGISTRY` in `ssl_tls_registry`. If you take this one then it'll be auto updated with the registry_fetch util.

    TLS_CERTIFICATE_TYPE = registry.TLS_CLIENTCERTIFICATETYPE_IDENTIFIERS_REGISTRY
    TLSCertificateType = EnumStruct(TLS_CERTIFICATE_TYPE)